backend/src/test/java/com/redhat/cloud/notifications/routers/UserConfigResourceTest.java

```diff
@@ -56,6 +68,9 @@ void beforeEach() {
     @RestClient
     TemplateEngineClient templateEngineClient;
 
+    @InjectSpy
```

Comment:
Do you need a Mockito spy for a specific reason? If not, then we usually use `@InjectMock`.